packages/aztec.js/src/setup/index.js

```diff
@@ -15,9 +15,10 @@ const fetch = require('cross-fetch');
 const bn128 = require('../bn128');
 
 const compressionMask = new BN('8000000000000000000000000000000000000000000000000000000000000000', 16);
-const POINTS_DB_URL = 'https://ds8m7zxw3jpbz.cloudfront.net/data';
 
-const setup = {};
+const setup = {
+    POINTS_DB_URL: 'https://ds8m7zxw3jpbz.cloudfront.net/data',
```

Comment:
We should do a proper environment file if we're going to go for a dynamic URL.

Reply:
Don't have time to do this right now, but agree this is something that should be done